foundation/settings.py

```diff
@@ -504,5 +502,8 @@ def _parse_email_list(varname):
 
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
 
+CAPTCHA_IMAGE_SIZE = (100, 50)
```

Comment:
```suggestion
CAPTCHA_IMAGE_SIZE = (200, 100)
```
This looks really small